server/api/baseApi.js

```diff
@@ -7,7 +7,7 @@ const getConfig = (requiresAuthorization) =>
     timeout: 10000,
     headers: {
       'Cache-Control': 'no-cache',
-      Authorization: requiresAuthorization ? `Bearer {${sessionStorage.getItem('token')}}` : '',
```

Comment:
The name was incorrect. See this: https://github.com/code4romania/monitorizare-vot-votanti-client/blob/d841cf81419d9f2b943c9b9be4f6d8b380a97e19/app/containers/App/reducer.js#L19